src/eos-shard-writer-v2.c

```diff
@@ -262,45 +262,80 @@ eos_shard_writer_v2_new_for_fd (int fd)
   return EOS_SHARD_WRITER_V2 (g_object_new (EOS_SHARD_TYPE_WRITER_V2, "fd", fd, NULL));
 }
 
+static int
+compress_blob_to_tmp (GInputStream *file_stream)
+{
+  char tmpl[] = "/tmp/shardXXXXXX";
```

Comment:
I could swear this goes to /tmp by default (otherwise it would make no sense to call it `mkstemp`) but I could be wrong - the docs seem light on explanations here.